src/unit_tests/wrappers/externals/sqlite/sqlite3_wrappers.h

```diff
@@ -36,6 +36,10 @@ void expect_sqlite3_bind_text_call(int position, const char *buf, int ret);
 int __wrap_sqlite3_bind_parameter_index(__attribute__((unused)) sqlite3_stmt * stmt,
                                         const char *zName);
 
+int __wrap_sqlite3_bind_double(__attribute__((unused)) sqlite3_stmt* stmt,
```

Comment:
Also here, check the parameters indentation

Reply:
Fixed!